includes/class-sensei-modules.php

```diff
@@ -1497,13 +1497,16 @@ public function admin_enqueue_styles() {
 	 * @return void
 	 */
 	public function course_modules_title() {
-
 		if ( sensei_module_has_lessons() ) {
-
-			echo '<header class="modules-title"><h2>' . esc_html__( 'Modules', 'sensei-lms' ) . '</h2></header>';
-
+			/**
+			 * Filters the module title on the single course page.
+			 *
+			 * @since 2.2.0
+			 *
+			 * @param string $html The HTML to be displayed.
```

Comment:
Could we add the Course or its ID as a parameter to this filter?

Reply:
Good call. Added in https://github.com/Automattic/sensei/pull/2737/commits/248ab26162414a84295784ade4138c11f851ea93.